### PR TITLE
Remove the Amalgamator from Event Settings and deprecate the file.

### PR DESCRIPTION
**Pull request by Camwyn**
Also removes the call in Main.php to listen for the keyword and process.

📷 [Bad](blob:https://theeventscalendar.atlassian.net/c1d1b8bb-366c-574c-af91-e991888874d9) `blob:https://theeventscalendar.atlassian.net/c1d1b8bb-366c-574c-af91-e991888874d9`
📸 [Good](https://d.pr/i/0NxbMF)

### CODE REVIEW

**Review by jesseeproductions — COMMENTED** [review]
@Camwyn - Looks good, fix the readme conflict and good to go.

**Comment by Camwyn** [comment]
@jesseeproductions can I get a re-review please? Updating dismissed your review.

**Review by jesseeproductions — APPROVED** [review]
@Camwyn Reapproved.

File: src/Tribe/Amalgamator.php [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?php
-
+_deprecated_file( __FILE__, 'TBD', 'No replacement.' );
```
Comment (bordoni) [inline src/Tribe/Amalgamator.php:2]: Move the file to the deprecated folder.

**Review by bordoni — CHANGES_REQUESTED** [review]
We have a depercated folder that we should use.

File: src/Tribe/Amalgamator.php (hunk 1 above)
Comment (Camwyn) [reply src/Tribe/Amalgamator.php:2]: Good call - thanks!

**Review by bordoni — DISMISSED** [review]

**Review by bordoni — APPROVED** [review]

**Pull request merged**